test/unified_format.py

```diff
@@ -1048,6 +1085,43 @@ def _testOperation_assertNumberConnectionsCheckedOut(self, spec):
         pool = get_pool(client)
         self.assertEqual(spec['connections'], pool.active_sockets)
 
+    def _testOperation_loop(self, spec):
+        global IS_INTERRUPTED
+        failure_key = spec.get('storeFailuresAsEntity')
+        error_key = spec.get('storeErrorsAsEntity')
+        successes_key = spec.get('storeSuccessesAsEntity')
+        iteration_key = spec.get('storeIterationsAsEntity')
+        if failure_key:
+            self.entity_map[failure_key] = []
+        if error_key:
+            self.entity_map[error_key] = []
+        if successes_key:
+            self.entity_map[successes_key] = []
+        if iteration_key:
+            self.entity_map[iteration_key] = 0
+        while True:
+            if IS_INTERRUPTED:
```

Comment:
Do we still need to catch KeyboardInterrupt?

Reply:
This is now for the interrupt_loop function that can be called by the workload executor.

Reply:
I'm not sure that answers my question. What happens when you crtl-C the tests during a loop op?

Reply:
Nothing appears to happen which is why I'm confused we got rid of that code that used the sigint to stop the loop.

Reply:
> Nothing appears to happen

Really? I'd expect ctrl-C to cause a KeyboardInterrupt to be raised from _testOperation_loop (even without using signals).

Reply:
It appears it was something with pycharm causing an issue. If I send it a ctrl-C through the command line it instantly exits.